docs/core/build/distribution-packaging.md

```diff
@@ -107,13 +107,15 @@ The following lists the recommended packages:
   - **Version:** \<runtime version>
   - **Example:** dotnet-sdk-2.1
   - **Contains:** (3),(4)
-  - **Dependencies:** `aspnetcore-runtime-[major].[minor]`, `dotnet-targeting-pack-[major].[minor]`, `aspnetcore-targeting-pack-[major].[minor]`, `netstandard-targeting-pack-[netstandard_major].[netstandard_minor]`, `dotnet-apphost-pack-[major].[minor]`, `dotnet-templates-[major].[minor]`
+  - **Dependencies:** `dotnet-runtime-[major].[minor]`, `aspnetcore-runtime-[major].[minor]`, `dotnet-targeting-pack-[major].[minor]`, `aspnetcore-targeting-pack-[major].[minor]`, `netstandard-targeting-pack-[netstandard_major].[netstandard_minor]`, `dotnet-apphost-pack-[major].[minor]`, `dotnet-templates-[major].[minor]`
+  - **Provides shared directory:** `{dotnet_root}/sdk`, `{dotnet_root}/sdk/NuGetFallbackFolder`
 
 - `aspnetcore-runtime-[major].[minor]` - Installs a specific ASP.NET Core runtime
   - **Version:** \<aspnetcore runtime version>
   - **Example:** aspnetcore-runtime-2.1
   - **Contains:** (6)
   - **Dependencies:** `dotnet-runtime-[major].[minor]`
+  - **Provides shared directory:** `{dotnet_root}/shared/Microsoft.AspNetCore.App`, `{dotnet_root}/shared/Microsoft.AspNetCore.All`
```

Comment:
I may be misunderstanding something At the bottom of this doc you mention the following, which seems to contradict the "Provides" section of `aspnetcore-runtime-[major].[minor]`.

> - The third-party libraries under `shared/Microsoft.AspNetCore.All` can't be easily built from source. So that folder is omitted from the `aspnetcore-runtime` package.

Reply:
I need to clarify in the doc what _Provides shared directory_ means. It's about the folder itself, and not the content under it. _Contains_ is about the content.

All folders listed as _Provides shared directory_ are folders which may be used by different packages. For example, aspnetcore-runtime-2.1 and aspnetcore-runtime-2.2 may both create the `{dotnet_root}/shared/Microsoft.AspNetCore.App` under which they place their own versioned folder.